Probesys/bileto
# new: Allow to import data from a ZIP archive

## Related issue(s)



https://github.com/Probesys/bileto/issues/560

## How to test manually



- reset the data with `make db-reset FORCE=true`
- create an archive matching the spec (sorry, too long to provide a working archive)
- run the command `./docker/bin/console app:data:import /path/to/archive.zip`
- login and check that data are present

## Checklist
